modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <transition
     :name="transitionType"
-    @enter="finishEnter"
-    @leave="finishExit"
+    @enter="onEnter"
+    @leave="onLeave"
```

Comment:
ℹ️ a bit of renaming, to avoid confusion with the `@after-enter` and `@after-leave` events.